examples/fnft_nsep_example.c

```diff
@@ -80,7 +83,7 @@ int main()
 
     /** Step 3: Call fnft_nsev and check for errors **/
 
-    int ret_code = fnft_nsep(D, q, T, &K, main_spec, &M, aux_spec, NULL, kappa,
+    int ret_code = fnft_nsep(D, q, T, phase_shift, &K, main_spec, &M, aux_spec, NULL, kappa,
```

Comment:
Here is another left-over `int` that should actually be an `FNFT_INT`.

Reply:
Fixed.